### DISCUSSION TITLE
I want to display type using customNode component

### DISCUSSION THREAD

**Discussion by Muhammad1Siddique** [xyflow/xyflow#4297] [Layouting]
![Screenshot from 2024-05-21 20-17-37](https://github.com/xyflow/xyflow/assets/48084054/81c066cf-4ab7-44f7-8320-768264129451)
{
        id: item.uuid,
        type: item.type,
        data: { 
          label: (
            <KubernetesIcon
              name={item.name}
              type={item.type}
              contextName={item?.contextName}
              namespace={item?.namespace}
              replicas={item?.replicas}
              phase={item?.phase}
              restartPolicy={item?.restartPolicy}
              serviceAccount={item?.serviceAccount}
              image={item?.image}
              podname={item?.podname}
              containername={item?.containername}
              readOnly={item?.readOnly}
              automountServiceAccountToken={item?.automountServiceAccountToken}
              creationTimestamp={item?.creationTimestamp}
            />
          ),
          expanded: true,
        }, 
        position: { x: 0, y: 0 },
      }
      i want to display type in the customNode component here is the code
      
      export default function CustomNode({ data, type }: NodeProps) {

  const label = getLabel(data);

  return (
    <section id='proflow' style={{zIndex: "10!important"}}>
      <div className={FlowStyle.node}>
        <div className={FlowStyle.label}>{data.label}</div>
        <Handle position={Position.Top} type="target" />
        <Handle position={Position.Bottom} type="source" />
        <div className={FlowStyle.button}>
          {type}
        </div>
      </div>
    </section>
  );
}
but I received custom again type object

**Comment by moklick** [comment]
Please try to format your issues so that we can read them https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax
